### ISSUE TITLE
Does the app support still images?

### ISSUE THREAD

**Issue by jagjordi** [theothernt/AerialViews#95]

**Comment by theothernt** [comment]
Currently, Aerial Views only supports video but I have started looking into supporting images.

There there any other image-only screensavers you've tried or are you looking to mix both videos and images in a screensaver?

**Comment by jagjordi** [comment]
I have tried others payed solutions. But I'm looking for an open source solution. Do you have any suggestions?

Best regards,

Jordi Altayó
________________________________

**Comment by theothernt** [comment]
I don't have any suggestions, but I'll update you when I make some progress with showing images.

**Comment by BenJamesAndo** [comment]
As a workaround I got an image and made it into a long .mp4 file. Native image support with the ability to specify how long each image shows for would be really nice. Then we could mix between videos and images. Handy for samba shares where we might want to dump videos and images from a recent trip for example.

**Comment by theothernt** [comment]
@BenJamesAndo that's a good solution in the meantime, did you use a video app or an online service to do that?

And to the bigger question, yes, I do hope to add support for images in future. Not sure on a timeline yet, but it may be a couple of months. I will keep this issue open and updated when I make progress.

**Comment by BenJamesAndo** [comment]
I just used Shutter Encoder. Any video editor would suffice just paste the still image into the timeline multiple times. There'd probably be some online services as well.

I had a thought that could make it even cooler. If the images did slow zooms in/out. That way there's always movement and the images won't look as super still compared to the videos. Then allow for choosing how long each image stays active.

**Comment by m0wer** [comment]
Any updates on this? How complex is the change (approximately)? Does anyone know any alternative apps for this functionality?

**Comment by theothernt** [comment]
> Any updates on this? How complex is the change (approximately)? Does anyone know any alternative apps for this functionality?

No updates yet I have a list of smaller features i'd like to implement first like more language support, weather and a few other overlays.

The change isn't too complex, it's more that it requires most of the app's code to be modified. Plus there is no 'player' for images, only video.

In the meantime, you could try [Photo Gallery and Screensaver](https://play.google.com/store/apps/details?id=com.furnaghan.android.photoscreensaver) ?

**Comment by fivestones** [comment]
I've tried Photo Gallery and Screensaver, and support for getting files from a network device seems to be broken. At least, I can't get it to work, and there are lots of reviews saying the developer seems to not respond to emails any more. I emailed him today about it, we'll see if he writes back. However, I'd much rather use AerialViews for this if I could.

**Comment by theothernt** [comment]
> I've tried Photo Gallery and Screensaver, and support for getting files from a network device seems to be broken. At least, I can't get it to work, and there are lots of reviews saying the developer seems to not respond to emails any more. I emailed him today about it, we'll see if he writes back. However, I'd much rather use AerialViews for this if I could.

I understand. Image support for Aerial Views is under way but I have been working on other features as well. I'm hoping to have image support before the end of the year but I'll update this issue/thread when there's progress.

**Comment by theothernt** [comment]
Also, I should mention - when it comes to images, these are the features I'm thinking about...

1. Fade in/out like videos (not configurable, will be in future)
2. Image changes every... 10sec, 30sec, 1min, 3min, 5mins
3. Slow image panning/zooming mode (aka. Ben Burns effect)
4. Vignette effect on/off (slight darkness around the edge of photos)

Any feedback or ideas?

**Comment by fivestones** [comment]
All of those sound good to me. It would be nice to have options for transitions between photos (fade in/out, just directly switch from one to the next, and maybe other nice looking transitions).

Another feature that is really important (maybe it goes without saying for you which is why you didn't mention it?) is for each image to get preloaded before it's displayed. I've seen other apps that get images from a network share, and they get to the point where one image is finished being shown, and then spent a bit of time on a loading screen while the next image is obtained from the network share. Obviously this is sub-optimal, and the previous image should continue to be shown until the new one is ready to show (even if for some reason it takes longer to get the image ready than the timeout of 10 seconds or whatever it's set to).

Another thing that was mentioned above by @theothernt is for it to be able to show a mix of videos and images. I've seen people post about other slideshow apps that when they were doing this, if the time per file was set to say 10 seconds, when a video was shown, it would only play the first 10 seconds. Or if the timeout per image was long, like 3 minutes, and the video was short, the video would end and nothing would happen until the 3 minutes were up. I think most people would want any video to play in its entirety and then immediately switch to the next file, regardless of the amount of time set to show each image.

**Comment by fivestones** [comment]
For situations with an overlay with other info on top of the image somewhere:

One other thing I've seen other places that looks good with images is to have a gradient of black over the image, starting maybe 3/4 of the way down, starting with 100% transparency and getting to maybe 50% transparency at the bottom. The look is kind of like a vignette effect but applied horrizontally only at the bottom of the image. Then overlays are put over the darkened area making it look great. I've seen this used in the brave browser new tab, and in other places too.

An example (looks like this one applies this kind of effect at both the top and bottom):
<img width="1116" alt="Screenshot 2023-10-26 at 3 33 48 PM" src="https://github.com/theothernt/AerialViews/assets/222087/b0d7d5cb-862c-48cb-9b9c-21215c9d89f4">

Here's the original image:
![image](https://github.com/theothernt/AerialViews/assets/222087/c89cbb9a-0ec3-49f0-97d6-ef18c4668ece)

**Comment by theothernt** [comment]
Thanks for the feedback. So, images are treated the same as videos in a number of ways.

I want the transitions between images and videos to be the same, ie. fading in and out - and there are some technical reasons why this probably won't change anytime soon (ie. different transition options)

Obviously, videos have different lengths where as images do not. Images will have their own timer, the default might be 30sec or 1min but you'll be able to change it.

Gradients, etc is something I'll be adding but it will be after the initial release of this feature. I'll have to see how development goes.

As for pre-loading. Currently there is none due to memory issues on most Android TV devices (the long story is that a pre-loading issue was why I forked Aerial Dream in the first place!). Images are tiny compared to videos so I don't see a need for pre-loading, especially from local/USB storage or SMB - loading even a 2MB image should be instant. Also, the image will only fade-in when the image has loaded, much like videos do know.

**Comment by fivestones** [comment]
As for pre-loading: how much of a limitation is this in android tvs? I think you're right about how there won't be a noticeable delay when loading from usb, but honestly one of the reasons I am hoping for photos to be implemented is that I can't find any other apps for android tv that show videos from a network store as a slideshow. It's certainly my use case--not from a usb drive. I guess what I'm saying is that if the android tv is able to play video where the entire screen contents are constantly changing, shouldn't they be able to preload one frame--a photo?
Maybe an alternative for photos coming from a network share is to copy the next photo to local storage while showing a different photo, then switch photos, then delete the previous photo.

**Comment by theothernt** [comment]
On Android TV, as opposed to Android phones, one of the main limitations is RAM. Plus, my app is not a normal app. It usually runs on-top of an existing app which is usually paused or suspended - then resumed when you exist the screensaver.

Playing video requires a lot of RAM as video is a) decompressed from the video file into larger pieces for actual playback and b) a buffer stores multiple pieces. So simply loading a 1080p video and pausing it requires at least 300MB - and the figure is a lot higher with 4K HDR video.

As for both images and video, I mask the initial loading by showing a black screen - and only after a) the video has loaded an initial amount or b) the image had loaded, do I fade out the black screen. As images are tiny compared to video, loading should be instant from USB storage or SMB - there will be no partial loading or jarring transition.

Also, for the purposes of loading an image, there wouldn't be much speed different between local storage vs SMB.

**Comment by brjhaverkamp** [comment]
Hello theothernt,

I came across your nice app when looking for a screensaver capable of accessing the photos/videos on my NAS.
I read here and in https://www.reddit.com/r/AndroidTV/comments/17eckeo/screensaver_that_can_pul
I hope you are still on that path. I would love to see that addition. It would make this the perfect screensaver app for my google TV.

Just out of curiosity, Can you update us on your plan and progres? No presure, I know this is open source and we should chip in if we are really desperate for it:-)

Kind regards,

Bert

**Comment by theothernt** [comment]
> Just out of curiosity, Can you update us on your plan and progres? No presure, I know this is open source and we should chip in if we are really desperate for it:-)

Hi Bert - no update just yet. I had planned to work on the app over the Christmas break but that didn't happen. I do hope to add image support in the next couple of releases - but I will post a test version here first. I expect to make progress over the next 2 or 3 weeks.

Also, thank you to everyone for their patience and input on the feature!

**Comment by theothernt** [comment]
Quick question for everyone - what images formats do you use - PNG, JPEG, HIEF, anything else?

**Comment by the-little-wave** [comment]
For images that would be accessed by Aerial Views it's currently all jpeg
for me. I use dng raw but can't see that ever needing to be read by this.

**Comment by BenJamesAndo** [comment]
I'd likely mostly use JPG or PNG. AVIF looks like it could get more popular in the future and support for WebP would be nice also.

**Comment by theothernt** [comment]
> For images that would be accessed by Aerial Views it's currently all jpeg for me. I use dng raw but can't see that ever needing to be read by this.

JPEG support will definitely be supported anyway.

**Comment by theothernt** [comment]
> I'd likely mostly use JPG or PNG. AVIF looks like it could get more popular in the future and support for WebP would be nice also.

AVIF, I'll add that to the list. It looks like it requires Android 12+ btw.

**Comment by bertremCA** [comment]
Sorry that I'm late to the party, but in response to your request for feature suggestions of October 25...
One additional feature I'd really love to see (and that I don't think has been suggested yet) would be some kind of tiled image display. This is one of the things that I really like about "Photo Gallery and Screensaver", even though that app is a piece of abominable crud in just about every other respect. It does do a nice job of showing a handful of images (up to about 7, i think) on the screen at the same time, and I like the "slide" transition that it uses for that. Looking forward to your beta!

**Comment by theothernt** [comment]
Here are two versions to test depending on your setup...

* [Aerial Views 1.6.4 beta1](https://www.dropbox.com/scl/fi/l8vko868odxu1fo3rcx94/aerial-views.1.6.4-beta1.apk?rlkey=kbe8suu7jim31z8xjc4hzfio2&dl=0) - this version will override the Play Store version so you'll have to enter all your settings again.
* [Aerial Views 1.6.4 beta1 debug](https://www.dropbox.com/scl/fi/yba7pcptxtkqkeh2c9d25/aerial-views.1.6.4-beta1-debug.apk?rlkey=ua46ua7e8qpi5ertn8x82m4jc&dl=0) - this version will install alongside any existing install of Aerial VIews

To use the image feature, look in...

1. _Settings > Video Sources > SMB/Samba Share > Choose media type >_ set to 'Images only' or 'Videos and images'
2. _Settings > Playlists >_ Slideshow speed and Image scaling options added

Displaying image only works for SMB, not local or USB devices - I'm hoping to add this in beta2 in the next week or so.

Please let me know if you find any issues or if existing features don't work as expected as many internal changes were made for this update.

**Comment by bertremCA** [comment]
Still so far so good with the beta!

And I thought of one other thing that would be really nice to have... captions read from EXIF data. The frustrating thing is that there seems to be little consensus about what field should be populated with these, but I personally ended up using "ImageDescription", though various online sources also suggest the use of "Description", "UserComment", and "XPTitle".  For videos, it seems even less clear, but I ended up settling on "Description" which at least VLC seems to recognize. As my Dad says... the great thing about standards is that you have so many to choose from.

**Comment by theothernt** [comment]
> And I thought of one other thing that would be really nice to have... captions read from EXIF data. The frustrating thing is that there seems to be little consensus about what field should be populated with these, but I personally ended up using "ImageDescription", though various online sources also suggest the use of "Description", "UserComment", and "XPTitle". For videos, it seems even less clear, but I ended up settling on "Description" which at least VLC seems to recognize. As my Dad says... the great thing about standards is that you have so many to choose from.

I just had a quick look and EXIF data extraction is supported on Android so it should be straight forward enough to add the feature in the next update or two.

My idea is to add the option to _Appearance > Location_ screen. There'll be an Images and Video section. In images, you can select which EXIF field will be used at the Location or Title.

Also, I hope to add the option for a gradient or background for the text displayed on the screen so it's easier to see. I notice that Google uses the gradient approach.

**Comment by burzka** [comment]
Oh, that is a really good feature. I am currently testing it out. My humble suggestion is a better way of handling vertical images. Google's idea of splitting the screen in two and showing two vertical photos together seems to be the best idea.

**Comment by theothernt** [comment]
Small update. I'll be releasing this build of the app in a couple of days, just waiting on some translations.

Testing did show a couple of bugs, one crash bug but I'm not sure if anyone noticed it, which have been fixed.

**Question** - roughly speaking, how many images are people using with the screensaver - a few hundred? a few thousand?   

Also, if the number is large, how long do you see "loading..." for?

**Comment by theothernt** [comment]
> That triggers a question, I just realize: Will the app be able to traverse subfolders for photos? My album hierarchy is Photalbum/decade/year/topic/photoxxx.jpg " So for instance Photo album Bert/2020s/2023/2023-01-01 New years party" Will that be picked up ?

Yes, the app will search subfolders - that feature was added a few releases ago. When you get the update, do let me know how long Aerial Views takes to start, I'm guessing it'll be several seconds or so.

**Comment by koolaidOHYeah** [comment]
This is so wonderful. Thank you for all your hard work. 
Great question! 

I have about 22,000 pictures
My "loading..." runs for about 50-60 seconds before the first picture shows. 
I would like to see this load much faster but I am happy to have something that works!

Subfolders work great.

**Comment by brjhaverkamp** [comment]
Ouch, 50-60 would be a bit much.  Hope there is some way Neil can make it to show the first image(s) before the total loading is ready:-) Even if those are only "semi random" or similar.

Good to hear the subfolders are no issue. Looking forward to testing. When are you releasing new version of the app?

**Comment by theothernt** [comment]
> I have about 22,000 pictures My "loading..." runs for about 50-60 seconds before the first picture shows. I would like to see this load much faster but I am happy to have something that works!

Thanks for that info. I have a few ideas on how to solve this issue but I'll test against 30,000 images.

**Comment by theothernt** [comment]
Ok, here is an update with a few fixes...

[Aerial Views 1.6.4 beta3](https://www.dropbox.com/scl/fi/kik1ouqcftkhofjy9u1kd/aerial-views.1.6.4-beta3.apk?rlkey=de9vqo43is0zuea9yi74zxtcy&dl=0)
[Aerial Views 1.6.4 beta3 debug](https://www.dropbox.com/scl/fi/t3aahpxd3p81t248kzsvi/aerial-views.1.6.4-beta3-debug.apk?rlkey=tweapzo833aboo61xc3b00gff&dl=0)

If you have 20k+ images, loading should be a lot faster. I have a lot of work to do in this area, but it'll happen over the next couple of releases. In the meantime, let me know if this helps...

**Comment by koolaidOHYeah** [comment]
Initial testing of Aerial Views 1.6.4 beta3 is awesome!
Loading time down to 5-7 seconds!!

**Comment by theothernt** [comment]
> @theothernt just wanted to say thanks for your hard work on this! I found the repo after some googling for a way to do video + photo screensaver on my Android TV. I'll try to install the beta and give any feedback. Is SMB currently working with the beta or still just local/USB?

Thanks! If you try beta 3, it supports both local storage and SMB for images. I'm planning on releasing this version in a few days but any feedback is welcome. After its release, I'll start talking/thinking about what features to add in the next couple of releases.

**Comment by bertremCA** [comment]
With ~9000 images, 1.6.4 beta3 has reduced the loading time from 34 seconds to 7 seconds. Much improved! Thanks!
I have yet to see any other issues with either beta1 or beta3

**Comment by curtisbarnard** [comment]
I installed beta and the following works great:
- Loading and playing images over SMB
- Loading and playing videos over USB

However Loading and playing videos over SMB is not working for me. It just sits with a loading screen forever. I've done the following to try and root cause:
- Verified with "Test Connection Settings" in Aerial Views that it sees the video files and selects for playback
- Confirmed that I can play video files over SMB on my desktop computer
- Confirmed that the same files can play in Aerial Views and Just Player when played via the USB connected to the TV

**Comment by curtisbarnard** [comment]
@theothernt The videos are one's that I own/recorded. I tried with three different MP4's from three different cameras as well as one webm that I downloaded from Youtube. 

The Apple and community videos seem to work fine when choosing 1080p H.264, 1080p HEVC or 4k HEVC. Also another data point is that this is a Sony XBR-75X90CH TV.

Switching to SMB v2 only or SMB v3 only doesn't seem to fix it either.

I also tried rebooting the TV just in case, but it still doesn't work.

I'm going to try installing the current release from the play store and see if it works. Will report back.

**Comment by theothernt** [comment]
> The Apple and community videos seem to work fine when choosing 1080p H.264, 1080p HEVC or 4k HEVC. Also another data point is that this is a Sony XBR-75X90CH TV.

Can you try [downloading a couple of videos from the Aerial Videos website](https://aerial-videos.netlify.app/) and see if they play from SMB?

> Switching to SMB v2 only or SMB v3 only doesn't seem to fix it either.

That's fine, it was a long shot.

> I also tried rebooting the TV just in case, but it still doesn't work.
> I'm going to try installing the current release from the play store and see if it works. Will report back.

I doubt either will do anything, but do let me know.

As you've noticed before, the issue is with video playback from SMB - but some video formats work, others don't. [There is another person with the same issue](https://github.com/theothernt/AerialViews/issues/128) although they have a Nvidia Shield, not a Sony TV.

**Comment by bertremCA** [comment]
FWIW, videos work fine for me with an nVidia Shield, though I don't have any 4K videos. Mine are mostly mp4's generated by my Pixel phone, typically at 1920x1080.

One other thing I noticed while testing videos though... a "center" option (as opposed to fill) like the one offered for images would be great. Vertical videos are cropped pretty badly.

**Comment by theothernt** [comment]
> FWIW, videos work fine for me with an nVidia Shield, though I don't have any 4K videos. Mine are mostly mp4's generated by my Pixel phone, typically at 1920x1080.

I don't know the cause of the video + Samba issue but it doesn't seem to happen to most people - but I'd still like to fix it, if possible. It's just frustrating that the same videos will play from one location but not another.

> One other thing I noticed while testing videos though... a "center" option (as opposed to fill) like the one offered for images would be great. Vertical videos are cropped pretty badly.

I've added that to the list, it should be possible in the next release or two.

**Comment by bertremCA** [comment]
> I don't know the cause of the video + Samba issue but it doesn't seem to happen to most people - but I'd still like to fix it, if possible. It's just frustrating that the same videos will play from one location but not another.

maybe a function of the samba server? i'm pulling from a server running FreeNAS v. 11.1-U7.

**Comment by curtisbarnard** [comment]
I'm running TrueNAS and was able to get a 1080p video to play, but not all 1080p videos to play. I moved the video discussion to #128 since it doesn't seem related to the work being done for the images.

**Comment by theothernt** [comment]
> maybe a function of the samba server? i'm pulling from a server running FreeNAS v. 11.1-U7.

That is possible but I think it's very unlikely, otherwise I would have seen more issues or crashes with different NAS devices.

(I use a Synology NAS btw)

**Comment by theothernt** [comment]
Version 1.7 has been released, I recommend updating if you're still running the beta...

https://github.com/theothernt/AerialViews/releases

Thanks to everyone who tested the feature and made suggestions.

**Comment by theothernt** [comment]
> Ha, I tried to update, but my Android TV can't find the update. Will try every hour for the next 24 hours ;-) 

If you're referring to the Play Store, then it won't be available for a few days. Typically, I release on a GitHub then wait about 5-7 days before posting it to the Play + Amazon Store.

> One small addition to your readme. For all linux adepts like me not be be feeling left out: I followed the windows instructions and it worked like a charm. Below my (anonymized) shell output:

Thanks for that, I'll update the instructions to add a) Linux user support and b) add a bit about "authorising the connection" the first time you connect to a device with ADB.

**Comment by brjhaverkamp** [comment]
So you haven't posted it on the play store yet?
That makes sense. I'll be patient. 
I thought I was looking at the delay between submitting and Googles verification. I read that can also take up to a few days.

**Comment by theothernt** [comment]
> So you haven't posted it on the play store yet? That makes sense. I'll be patient. I thought I was looking at the delay between submitting and Googles verification. I read that can also take up to a few days.

Due to the limited amount of testing I can do with different devices, I stagger the roll out of updates.

Sometimes there is a beta for certain features. Then a stable release on GitHub. 5-7 days later on the Play Store - 20% rollout for 1 to 2 days then 100%.

Review times in the Play Store can vary. For the first year, it was 3-4 days I think. Now it's 1 hour or less.

**Comment by brjhaverkamp** [comment]
Hello Neil,

I upgraded a few days ago, (thanks for the heads-up:-) )
And I am happy to report that the screensaver is working niceley! Huray and thank you soo much!

Few observations: 
First it took me a while to figure out that  you have to select "Media type" under "Network". Images only/ video only/ videos &images. Wouldn't it be more logic to put that under  "Appearance" ? It is not a source specific setting, right?

Secondly, I after I initially set aerialview as my screensaver, I couldn't figure out why it started almost immediately.
Then I reread the instructions: _To change the default timeout use this command with a value in milliseconds. So, 5 minutes is 30000, 10 minutes is 60000 and so on._
I had read over it without thinking. But 30000ms is 30 seconds. And that was exactly what I was seeing :-)

Can you change this in the documentation? so: _To change the default timeout use this command with a value in milliseconds. So, 5 minutes is 600000, 10 minutes is 1200000 and so on._

And then a feature request: I would like to see the folder name of the photo in the left bottom. As I mentioned earlier I have sorted my photos as in Photalbum/decade/year/topic/photoxxx.jpg
" So for instance Photo album Bert/2020s/2023/2023-01-01 New years party"
So the folder the photo or video is in is the topic name. If  I could show that name as "title" that would be great. Or is this already possible somehow?

In all great work! Thanks for the great app.

Bert

**Comment by theothernt** [comment]
> Few observations: First it took me a while to figure out that you have to select "Media type" under "Network". Images only/ video only/ videos &images. Wouldn't it be more logic to put that under "Appearance" ? It is not a source specific setting, right?

Currently, Aerial Views is marketed as a video screensaver, so that is the default. In future, on first run of the app, I will probably give the user the choice of videos, pictures or both.
 
> Can you change this in the documentation? so: _To change the default timeout use this command with a value in milliseconds.

Thanks for catching that mistake, it's fixed now.

> And then a feature request: I would like to see the folder name of the photo in the left bottom. As I mentioned earlier I have sorted my photos as in Photalbum/decade/year/topic/photoxxx.jpg " So for instance Photo album Bert/2020s/2023/2023-01-01 New years party" So the folder the photo or video is in is the topic name. If I could show that name as "title" that would be great. Or is this already possible somehow?

That might be possible but I would need a few more examples so see how that might be formatted.

**Comment by brjhaverkamp** [comment]
Hi Neil, 

What examples do you mean?
Hope this helps: 
As I mentioned the last level subfolder name is the topic of the set of photo in that folder.
That foldername is a string of typically 10-30 characters. 
e.g. "Christmas 2024", "2023-12-05 St Nicolas" or "2021-02-18 Birthday" 
I normally put a date and a few words in the folder name.
If it could  be shown in the bottom of the image, that would be simple and effective.
I've seen that in other screensavers. e.g. in Kodi. 

Regards,

Bert

**Comment by theothernt** [comment]
>  That foldername is a string of typically 10-30 characters. e.g. "Christmas 2024", "2023-12-05 St Nicolas" or "2021-02-18 Birthday" I normally put a date and a few words in the folder name. If it could be shown in the bottom of the image, that would be simple and effective.

I had a quick look at the code for the Kodi screensaver you mentioned. If you have the following folders...

_/photos/Christmas 2024/Photo.jpg
/photos/2023-12-05 St Nicolas/Photo.jpg
/photos/2021-02-18 Birthday/Photo.jpg_

I think it will display...

_Christmas 2024 / Photo
2023-12-05 St Nicolas / Photo
2021-02-18 Birthday / Photo_

If that's the case, that is something I can add to Aerial Views.

**Comment by theothernt** [comment]
> What screensaver in Kodi did you look at? I used the Picture Slideshow Screensaver add-on. In this screensaver you can set (Under the "Addtional" tab) the Display image name.

I think that was it, I did only look at the code though.

> * Disabled
> * Use full path
> * Use folder- and filename
> * Use foldername
> * Use filename

I can implement all of those. I'm just not sure if it'll be in 1.7.1 or 1.7.2 yet.

**Comment by brjhaverkamp** [comment]
Hi Neil, 
That would be great!! I'm already enjoying the current version very much. So I'll wait (im)patiently :-)
I just donated a small amount to your coffee fund for your efforts.

Regards,

Bert

**Comment by brjhaverkamp** [comment]
Hi Neil,

I just wanted to check if you are closer to implementing showing the the name or title, as we discussed a while back.
Reason is that my kids mentioned it this evening when the screensaver was popping up. They remembered it from the previous osmc/kodi system. So now the whole family supports the request apparently :-)

Regards,

Bert

**Comment by brjhaverkamp** [comment]
Ah, forgot to mention: In the field of an "not so serious feature request" when I told the kids that we had already discussed it here, my youngest though it is a very good idea if you could delay the title by a few seconds , so everyone could first guess what the photo was about. As a quiz or game :-)

**Comment by theothernt** [comment]
> I just wanted to check if you are closer to implementing showing the the name or title, as we discussed a while back. 

I should have a beta version to try in a few days.

It's not likely I'll be able to add a 'delay' option to the photo/video descriptions any time soon - but I may be adding a feature to hide all overlays until a up/down button is pressed - that may help?

**Comment by theothernt** [comment]
> A beta in a few days? That is great news! 

Here is [Aerial Views v1.7.1 beta1](https://www.dropbox.com/scl/fi/23zas2ltfae8e807zht2p/aerial-views-1.7.1-beta1.apk?rlkey=nl5pxgsfmgwwdo9q77euv7dsb&dl=0). A couple of features to be aware of...

1. Settings > Appearance > Location / Description > New formatting options for photos, videos, etc
2. Settings > Appearance > (scroll down) Gradients

**Comment by brjhaverkamp** [comment]
I just received my update via the play store. It works great. Thanks for creating the perfect screensaver for me :-)
